pom.xml

```diff
@@ -40,9 +40,11 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
+        <ahc.version>2.10.4</ahc.version>
```

Comment:
do we need this version or can we inherit from camel-ahc ?

Reply:
Actually not for this PR directly, but it will be needed for the Telegram extension that needs AHC but has nothing to do with camel-ahc.

Reply:
yeah but this add some maintenance cost as we need to be sure this is in sync with the version used by camel

Reply:
Yes, that could be mitigated by 

(a) Importing the Camel BoM, or 
(b) Enabling the Dependency Convergence Rule on the Maven Enforcer plugin or
(c) A groovy script :)

Reply:
I think we really need to consider option 1, eventually we may need to clean up a bit the camel's bom but I think that as long term we'll end up supporting most of the component camel supports and the chance to hit this issue again is in my opinion high